src/components/app.js

```diff
@@ -0,0 +1,11 @@
+import Searchbar from './searchbar';
+
+
+function App() {
+  return (
+    Searchbar()
```

Comment:
It probably makes most sense to render `CityWeather` as a sibling to `Searchbar` since `Searchbar` includes more than just the searchbar - it's a bit of a misnomer